src/refmt/refmt_impl.ml

```diff
@@ -92,9 +92,12 @@ let refmt
     | _ -> `Ok (List.iter (fun file -> refmt_single (Some file)) input_files)
   with
   | Printer_maker.Invalid_config msg -> `Error (true, msg)
```

Comment:
@jordwalke re: your question on Discord: I don't think we throw any other exceptions here, and therefore it is OK to only catch these.

Feel free to prove me wrong and I'll update the code accordingly.